aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -40,11 +41,18 @@
     + r"Memory\s+Size:\s+(?P<{}>\d+)\s+MB\s+".format(MEMORY_ALLOCATED_FIELD_NAME)
     + r"Max\s+Memory\s+Used:\s+(?P<{}>\d+)\s+MB".format(MAX_MEMORY_USED_METRIC_NAME)
 )
-
-# Pull memorysize tag and cold start from report
-TAGS_TO_PARSE_FROM_REPORT = [
-    MEMORY_ALLOCATED_FIELD_NAME,
-]
+# Make separate regex to account for cold start
+REPORT_LOG_REGEX_INIT = re.compile(
```

Comment:
Any reason why we can't combine this with `REPORT_LOG_REGEX` by marking "Init Duration" being optional?

Reply:
No. I'll update it since that's a better solution.